config/kernel/linux-mx7dsabre-default.config

```diff
@@ -0,0 +1,5134 @@
+#
```

Comment:
Try to resolve Docker kernel dependencies with help of this script: https://raw.githubusercontent.com/docker/docker/master/contrib/check-config.sh